praktikumsplaner-frontend/frontend/src/composables/formatter.ts

```diff
@@ -11,5 +11,9 @@ export function useFormatter() {
         return date ? moment(date).format("L") : "";
     }
 
-    return { startingCharUpperCase, formatDate };
+    function formatDateFromString(date: string): string {
```

Comment:
```suggestion
    function formatDateFromString(date: string | undefined): string {
```
this should make pass the frontend build